coroutines-extra/src/test/scala/org/coroutines/extra/async-await-tests.scala

```diff
@@ -1,56 +1,29 @@
-package org.coroutines
+package org.coroutines.extra
```

Comment:
It seems that the async await tests are still moved - we should keep them in the base project.